Harmony/Documentation/articles/patching-troubleshooting.md

```diff
@@ -0,0 +1,19 @@
+## Troubleshooting
+
+### Dll
+
+- make sure you're actually running the built DLL you expect you are (post-build copy problems)
```

Comment:
wording: more like "that the harmony dll you compile against is already loaded" or so